ownCloudAppShared/Client/Sharing/ShareViewController.swift

```diff
@@ -259,7 +259,18 @@ open class ShareViewController: CollectionViewController, SearchViewControllerDe
 
 	open override func viewDidLoad() {
 		super.viewDidLoad()
-
+		
+		let secureView = SecureTextField().secureContainerView
```

Comment:
The suggestions about unifying the code to inject a secure container that I made in `BrowserNavigationViewController.swift` would also apply here.